samples/Mvc.Server/Startup.cs

```diff
@@ -46,6 +47,18 @@ public void ConfigureServices(IServiceCollection services)
                 options.ClaimsIdentity.RoleClaimType = Claims.Role;
             });
 
+            // OpenIddict offers native integration with Quartz.NET to perform scheduled tasks
+            // (like pruning orphaned authorizations/tokens from the database) at regular intervals.
+            services.AddQuartz(options =>
+            {
+                options.UseMicrosoftDependencyInjectionJobFactory();
+                options.UseSimpleTypeLoader();
+                options.UseInMemoryStore();
+            });
+
+            // Register the Quartz.NET service and configure it to block shutdown until jobs are complete.
+            services.AddQuartzHostedService(options => options.WaitForJobsToComplete = true);
```

Comment:
For use case where there would be multiple scheduler, we would need an overload or something to resolve specific scheduler, just commenting on Quartz logic, not your implementation 🙂

Reply:
Yep. Since MS's DI doesn't support named services, it's something you'd have to implement yourself, likely by using derived scheduler types. Named options w/ `Microsoft.Extensions.Options` would be a very nice alternative if you wanted to support multiple schedulers and different options per scheduler.